src/main/java/edu/ucsb/cs56/ucsbapi/academics/curriculums/v1/classes/Course.java

```diff
@@ -27,14 +27,23 @@ public String getQuarter() {
         return this.quarter;
     }
 
+    public String getQuarterNoSpace() {
+        return this.quarter.replaceAll("\\s","");
+    }
+
```

Comment:
@btk5h observed during our joint code review that since `getQuarterNoSpace` and `getCourseIdNoSpace` are always called adjacent and then concatenated, maybe consider replacing with a single method that returns a `courseKey` maybe `getCourseKey`?

Reply:
Good idea! I will get on that.

Reply:
Updated!